gulp-tasks/puppeteer/build_docs.js

```diff
@@ -59,11 +59,11 @@ async function generateTOC(tocYaml, tags) {
       title: tag,
       section: [{
         title: 'Docs',
-        path: `${GITHUB_PROJECT_URL}/blob/${tag}/docs/api.md`,
+        path: `https://pptr.dev/#?product=Puppeteer&version=${tag}&show=outline`,
         status: 'external',
       }, {
         title: 'Release Notes',
-        path: `${GITHUB_PROJECT_URL}/releases/tag/${tag}`,
```

Comment:
is the `${GITHUB_PROJECT_URL}` variable used any more?

Reply:
Yea, for the old releases.